packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -60,7 +60,7 @@ export const floatingLocalVideoModalStyle = (
         boxShadow: theme.effects.elevation8,
         ':focus-within': {
           boxShadow: theme.effects.elevation16,
-          border: `2px solid ${theme.palette.neutralPrimary}`
+          border: `0.125rem solid ${theme.palette.neutralPrimary}`
```

Comment:
there is a utility function `_pxToRem` in common package. (In case you want to use that :))